src/adapter/ads/nativeads.js

```diff
@@ -28,24 +28,26 @@ let NativeAdsAdapter = youbora.Adapter.extend({
   /**  @returns {String} - current ad position (only ads) */
   getPosition: function () {
     let returnValue = youbora.Adapter.AdPosition.MIDROLL;
-    switch (this.adPosition) {
-      case 'preroll':
-        returnValue = youbora.Adapter.AdPosition.PREROLL;
-        break;
-      case 'postroll':
-        returnValue = youbora.Adapter.AdPosition.POSTROLL;
-        break;
-      case 'midroll':
-        break;
-      case 'overlay':
-        returnValue = 'overlay';
-        break;
-      default:
-        if (!this.plugin.getAdapter().flags.isJoined) {
+    if (this.adPosition) {
```

Comment:
I think this if-else logic is equal to the switch-default case so you can revert it.